master_server/dummy_vm_server.py

```diff
@@ -103,7 +112,14 @@ def is_active():
 def flag_status():
 #  Returns the state of VM
   return {'status': flag}
-
+  
+@APP.route('/get_status', methods=['GET', 'POST'])
+def get_status_of_request():
+  task_status_request = Request_pb2.TaskStatusRequest()
+  input_file = request.files['task_request']
```

Comment:
Are we passing the proto as files?

Reply:
Yes,we are passing the proto as files

Reply:
Yes,we are still passing it as files,as we discussed in our regular meet.